ui/js/resetpassword.js

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright 2014 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+require(['jquery','oae.core'], function($, oae) {
+
+    // Set the page title
+    oae.api.util.setBrowserTitle('Reset Password');
+
+    /**
+     * ResetPassword: Last Step for reseting passord
+     */
+	  var resetPassword = function(username,secret){
+  		$(document).on('submit','#resetpassword-input-password', function(event){
+  			
+  		// disable redirect to other page
+  		event.preventDefault();
+  			
+  		var password = $.trim($('#resetpassword-new-password', $(this)).val());
+  		
+          $.ajax({
+              'url': '/api/auth/local/reset/change/' + username,
+              'type': 'POST',
+              'data': {'newPassword':password, 'secret':secret},
+              'success': function(data) {
+                  //callback(null, data);
+  				        oae.api.util.notification('Congratulations!','Your password has been succesfully changed!');
+  				
+          				// wait for 3 seconds and redirect the user to mainpage
+          				setTimeout(
+              				  function() 
+              				  	{
+              				    	//redirect the user to setting page
+              						  window.location.href = "/me";
+              						
```

Comment:
This no longer works due to changes to the rest of the UI... I think we should just send them to the front page anyway.